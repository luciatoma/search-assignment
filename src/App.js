import React from 'react';
import Home from './components/pages/Home';

const App = () => (
    <div>
        <Home />
    </div>
);

export default App;
