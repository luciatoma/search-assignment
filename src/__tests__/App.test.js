import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { MockedProvider } from 'react-apollo/test-utils';

import App from '../App';

/* global document */

it('renders without crashing and matches snapshot', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <MockedProvider>
            <App />
        </MockedProvider>,
        div
    );
    ReactDOM.unmountComponentAtNode(div);
    const tree = renderer
        .create(
            <MockedProvider>
                <App />
            </MockedProvider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
