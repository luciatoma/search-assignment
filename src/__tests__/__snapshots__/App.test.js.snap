// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<div
  className="sc-jzJRlG bBVbYm"
  onScroll={[Function]}
>
  <div
    className="sc-cSHVUG kmYqSE"
  >
    <div
      className="sc-bdVaJa cpqOBv"
    >
      <input
        className="sc-bwzfXH erGLxu"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search..."
        type="text"
        value=""
      />
      <img
        alt="Search"
        className="sc-htpNat NRooi"
        onClick={[Function]}
        src="magnifying-glass.svg"
      />
      
    </div>
  </div>
</div>
`;
